test/ui-test/src/screens/StatusCommunityScreen.py

```diff
@@ -93,12 +93,36 @@ def _find_category_in_chat(self, community_category_name: str):
                 return True, item
         return False, None
 
-    def open_edit_channel_popup(self):
+    def _toggle_channels_in_category_popop(self, community_channel_names: str):
+        for channel_name in community_channel_names.split(", "):
+            [loaded, channel] = self._find_channel_in_category_popup(channel_name)
+            if loaded:
+                click_obj(channel)
+            else:
+                verify_failure("Can't find channel " + channel_name)
+
+    def _get_checked_channel_names_in_category_popup(self):
+        listView = get_obj(CreateOrEditCommunityCategoryPopup.COMMUNITY_CATEGORY_LIST.value)
+        result = []
+
+        for index in range(listView.count):
+            listItem = listView.itemAtIndex(index)
+            if (listItem.checked):
+                result.append(listItem.objectName.toLower())
+        return result
+
+    def _open_edit_channel_popup(self):
         StatusMainScreen.wait_for_banner_to_disappear()
 
         click_obj_by_name(CommunityScreenComponents.CHAT_MORE_OPTIONS_BUTTON.value)
         click_obj_by_name(CommunityScreenComponents.EDIT_CHANNEL_MENU_ITEM.value)
 
+    def _open_category_edit_popup(self, category):
+        # For some reason it clicks on a first channel in category instead of category
+        squish.mouseClick(category.parent, squish.Qt.LeftButton)
```

Comment:
Please use our wrapper, `SquishDriver` here, instead of direct use of `squish` in screens layer.